validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -153,4 +153,24 @@ public void extra_config() {
     untilLogged(SYSTEM_CONFIG_APPLY, SYSTEM_CONFIG_APPLY_LEVEL);
   }
 
+  @Test(timeout = ONE_MINUTES_MS)
+  @Feature(stage = BETA, bucket = SYSTEM)
+  @Summary("Check that the device publishes minimum required log entries when receiving config")
+  public void config_logging() {
+    deviceConfig.system.min_loglevel = Level.DEBUG.value();
```

Comment:
how is this test functionally different than what's covered already?  Some of the other tests essentially do exactly this along with some other stuff... ?  Each tests adds an overhead, so just wondering what the underlying motivation is...

Reply:
They aren't really, `extra_field` and `broken_config` do both currently test for the exact same config logging, but as a  dependancy upon those behaviours for testing additional required behaviours. 

There isn't anywhere to say that as a basis "when a device receives a config message it should publish receive, parse, apply system logentries" which is needed. Although, I didn't include anything about the status block

The motivation is better information for both developers and report consumers